### ISSUE TITLE
Question about Automatic Loss Balancing Function

### ISSUE THREAD

**Issue by roborocklsm** [Zhongdao/Towards-Realtime-MOT#172]
Hey, thanks for your great work!
But I have a question about  the **Automatic Loss Balancing Function**.

According to [_Kendall, Gal, andCipolla 2018_](https://arxiv.org/pdf/1705.07115.pdf), **continuous output** and **softmax output** have different coefficients: continuous one contains 1/2 while softmax does not. But I notice both in your paper and code give coefficient 1/2 to all three losses.

Would you mind giving some further explanations?

**Comment by roborocklsm** [comment]
OKAY, I find out the loss function is for logits but not for the probability after softmax. 

Have a nice day.

**Comment by Zhongdao** [comment]
Yes, there should be a 1/2 in softmax according to the derivation. But in practice, we find this constant factor hardly influence the final performance, so we remove the 1/2.

**Comment by roborocklsm** [comment]
@Zhongdao Yes, sure, you are right. I have tried and it does have no difference between two results.

Thx.
